content/docs/concepts/policies-and-telemetry/index.md

```diff
@@ -44,7 +44,7 @@ At a high level, Mixer provides:
 Beyond these purely functional aspects, Mixer also has [reliability and scalability](#reliability-and-latency) benefits as outlined below.
 
 Policy enforcement and telemetry collection are entirely driven from configuration.
-It's possible to [completely disable these features](/docs/setup/kubernetes/install/minimal/)
+It's possible to completely disable these features by installing the [minimal Istio profile](/docs/setup/kubernetes/install/helm/)
```

Comment:
note: mixer policy is off by default.  we don't necessarily need to refer users to minimal profile here.

Reply:
I'll reword this part.